frontends/web/src/routes/account/send/send.tsx

```diff
@@ -115,7 +115,7 @@ class Send extends Component<Props, State> {
     isAborted: false,
     isUpdatingProposal: false,
     noMobileChannelError: false,
-    fiatUnit: fiat.state.active,
+    fiatUnit: this.props.activeCurrency,
     coinControl: false,
     btcUnit : 'default',
```

Comment:
question: should `btcUnit` come from context in the future (maybe once send is functional component)?

Reply:
good point, I think we could use context. Just a bit strange that we use config here and context on `rates.tsx`. Which is the same as the old implementation (config on send but store on `rates`).

We set the value of `btcUnit` in context from config too, so essentially it's the same thing..

Do you think that's intentional? 🤔

Reply:
> Do you think using config on send and context/store on rates is intentional? 🤔

I believe btcUnit was added later, but send component was not updated to use it from the custom store.

Lets keep as is and once we have send as functional change it so it doesn't need to get the config, but it can take if from context.